tartufo/scanner.py

```diff
@@ -692,27 +743,6 @@ def load_repo(self, repo_path: str) -> pygit2.Repository:
             config_file = None
         if config_file and config_file != self.global_options.config:
             self.config_data = data
-            include_patterns = list(data.get("include_path_patterns", ()))
-            repo_include_file = data.get("include_paths", None)
-            if repo_include_file:
-                repo_include_file = pathlib.Path(repo_path, repo_include_file)
-                if repo_include_file.exists():
-                    with repo_include_file.open() as handle:
-                        include_patterns += handle.readlines()
-            if include_patterns:
-                include_patterns = config.compile_path_rules(include_patterns)
-                self._included_paths = list(set(self.included_paths + include_patterns))
-
-            exclude_patterns = list(data.get("exclude_path_patterns", ()))
-            repo_exclude_file = data.get("exclude_paths", None)
-            if repo_exclude_file:
-                repo_exclude_file = pathlib.Path(repo_path, repo_exclude_file)
-                if repo_exclude_file.exists():
-                    with repo_exclude_file.open() as handle:
-                        exclude_patterns += handle.readlines()
-            if exclude_patterns:
-                exclude_patterns = config.compile_path_rules(exclude_patterns)
-                self._excluded_paths = list(set(self.excluded_paths + exclude_patterns))
```

Comment:
With this code block gone, how do these supplemental patterns now get added to the config?

Reply:
I think L760 is handling this, although I'm fuzzy on why the config_data property and setter were removed just above (or alternatively why they existed previously).

Reply:
There is another config_data property at line 244. So I am removing the duplicate one here.